XcodeAutoBasher/XcodeAutoBasher/Models/VOKProjectContainer.m

```diff
@@ -116,6 +116,52 @@ - (VOKDirectoryWatcher *)directoryWatcher
     return _directoryWatcher;
 }
 
+- (NSDictionary *)environmentVariables
+{
+    if ([NSThread isMainThread]) {
+        NSLog(@"VOKProjectContainer environmentVariables being retrieved on the main thread.");
```

Comment:
Maybe.  It was convenient for it not to assert during development, but I'm not sure I can justify it not asserting now... well, except that even if it asserts, the assert won't make it to release builds, but :shrugs:.